src/pages/signup/StepThree.jsx

```diff
@@ -8,71 +8,73 @@ import Button from '@mui/material/Button';
 
 export default function StepThree() {
   return (
-    <div>
-         <FormControl fullWidth>
+    <div className="StepThreeForm">
+        <FormControl sx={{width: 400}} className="StepThreeInput">
             <InputLabel>Gender</InputLabel>
             <Select
                 labelId="demo-simple-select-label"
                 id="demo-simple-select"
```

Comment:
for the ids on these components, remember with HTML they will need to be unique. We could probably rename them more clearly (without the word `demo`) ?